sdks/python/setup.py

```diff
@@ -186,7 +186,7 @@ def get_version():
     'google-auth>=1.18.0,<2',
     'google-cloud-datastore>=1.8.0,<2',
     'google-cloud-pubsub>=0.39.0,<2',
-    'google-cloud-bigquery-storage>=2.4.0',
```

Comment:
I think we also need to add this dependency to https://github.com/apache/beam/blob/master/sdks/python/container/base_image_requirements.txt

Reply:
Done